edb/lang/edgeql/compiler/clauses.py

```diff
@@ -71,6 +75,30 @@ def compile_orderby_clause(
                 ir_sortexpr.context = sortexpr.context
                 stmtctx.enforce_singleton(ir_sortexpr, ctx=exprctx)
 
+                # check that the sortexpr type is actually orderable
+                sort_type = inference.infer_type(ir_sortexpr, exprctx)
+                # Verify that '>=' is defined for 2 sort_type
+                # expressions. We rely on comparison operators to be
+                # defined in a way that makes it look like PostgreSQL
+                # is using '>=' or '<=' in its internal ORDER BY
+                # clause.
+                env = exprctx.env
+                opers = env.schema.get_operators(
+                    '>=', module_aliases=exprctx.modaliases)
```

Comment:
Let's follow the PostgreSQL behavior of using the `<` operator for ascending ordering and `>` for descending.

Reply:
Updated.